go.mod

```diff
@@ -1,44 +1,44 @@
 module github.com/zitadel/zitadel
 
-go 1.23.4
+go 1.23.6
```

Comment:
do we upgrade to 1.24 seperately?

Reply:
also since we then can use the `tool` directive

Reply:
Already first report that OIDC fails to build with go 1.24: https://github.com/zitadel/oidc/issues/715. That bug is probably related to the [FIPS rollout](https://go.dev/doc/security/fips140) of Go, which changes some behavior on crypto Random read. 

IMO we should update our ecosystem first (SAML, OIDC, Passwap) and if all works as intended, bump Zitadel.